apps/mull-api/src/app/event/event.resolver.ts

```diff
@@ -86,6 +86,6 @@ export class EventResolver {
 
   @ResolveField(/* istanbul ignore next */ () => Media)
   async image(@Parent() event: Event) {
-    return await this.eventService.getEventImage(event.id);
+    return this.eventService.getEventImage(event.id);
```

Comment:
I wasn't aware that `async` wraps returns with a promise for us and we don't have to explicitly return a promise. Good to know!